### DISCUSSION TITLE
Blog "Status Update" - PeerReviews

### DISCUSSION THREAD

**Discussion by qvalentin** [FileFighter/filefighter.github.io#38] [General]
Leave your feedback about our new blog post below :D

**Comment by SuperLonci** [comment]
Hey Filefighters,
One can see that you have already progressed very far and I am very excited to see the finished project. 
Regarding your blog post I would love to see an more details in your uml diagram. You can select the details to be displayed in the upper part of the intellij window. 
Apart from that thumbs up! And i have to say that I really like your Logo!  

Stay fit, 
your FitCom Team

**Comment by tomizius** [comment]
Hello FileFighter,

it sounds like you are really good on track.
I think it's great that you have already implemented three things. The picture also looks great.

Thumbs up from the traewellers.
Cheers and happy coding
